wrangler-service/src/main/java/io/cdap/wrangler/service/directive/AbstractDirectiveHandler.java

```diff
@@ -181,11 +198,19 @@ protected DirectiveExecutionResponse generateExecutionResponse(
             type = type.substring(0, 1).toUpperCase() + type.substring(1).toLowerCase();
           }
           types.put(fieldName, type);
-          if ((object.getClass().getMethod("toString").getDeclaringClass() != Object.class)) {
-            value.put(fieldName, object.toString());
+
+          if ((object instanceof StructuredRecord)
```

Comment:
under what condition can this object be `StructuredRecord`? It is not possible that it is still a StructuredRecord right?